eng/Directory.Build.Data.props

```diff
@@ -55,6 +55,7 @@
       ($NoWarn);
```

Comment:
this line looks wrong

Reply:
I know it looks duplicated with CA1308  above, but without it, build fails.

Reply:
No, the syntax is wrong, it should be `$(NoWarn);` instead